app/app/src/main/java/edu/osu/urban_security/security_app/SignInActivity.java

```diff
@@ -147,11 +147,11 @@ private void onAuthSuccess() {
             @Override
             public void onDataChange(DataSnapshot dataSnapshot) {
                 DataSnapshot childSnapshot = dataSnapshot.child("users");
-                User userObj = childSnapshot.child(mAuth.getUid()).getValue(User.class);
```

Comment:
Lost getUid()  method when user was switched from type FirebaseUser to User. Temporary fix is to comment them out.

See commit 52e181c7813e72dfb85c0146810bb70c504c6cd0